include/swift/Runtime/CrashInfo.h

```diff
@@ -18,22 +18,52 @@
 #ifndef SWIFT_CRASHINFO_H
 #define SWIFT_CRASHINFO_H
 
-#include "SwiftStdint.h"
+#include <inttypes.h>
 
 #ifdef __cplusplus
 namespace swift {
+namespace runtime {
+namespace backtrace {
 extern "C" {
 #endif
 
 struct CrashInfo {
```

Comment:
Ditto, a doc comment would help to clarify what `mctx` is, while for other fields additional context would also be helpful.